simapp/helpers.go

```diff
@@ -31,3 +38,68 @@ func Setup(isCheckTx bool) *SimApp {
 
 	return app
 }
+
+// GenTx generates a signed mock transaction.
+func GenTx(msgs []sdk.Msg, accnums []uint64, seq []uint64, priv ...crypto.PrivKey) auth.StdTx {
```

Comment:
We should add a comment at the top of this file that these helpers are only meant for testing and are not required on every SDK app

Reply:
I renamed the file to `test_helpers.go`. Thoughts on that approach vs adding a comment and leaving as `helpers.go`?

Reply:
I guess it's fine with `test_helpers.go`